content/docs/introduction/getting_started.md

```diff
@@ -160,6 +160,7 @@ processes:
 # Fetch the client library code and compile example.
 git clone https://github.com/prometheus/client_golang.git
 cd client_golang/examples/random
+go get
```

Comment:
Hmm, shouldn't we assume the user has a proper Go setup? Then it would really just be `go get`.